homeassistant/components/sms/const.py

```diff
@@ -27,3 +36,61 @@
     {"value": "76800", "label": "76800"},
     {"value": "115200", "label": "115200"},
 ]
+
+SIGNAL_SENSORS: Final[dict[str, SensorEntityDescription]] = {
+    "SignalStrength": SensorEntityDescription(
```

Comment:
For a future PR, you can add state_class=MEASUREMENT for this and others so stats are being generated